projects/plugins/jetpack/extensions/blocks/calendly/calendly.php

```diff
@@ -112,9 +112,19 @@ function load_assets( $attr, $content ) {
 			esc_attr( $block_id )
 		);
 		$script  = <<<JS_END
-jetpackInitCalendly( '%s', '%s' );
+jetpackInitCalendly( %s, %s );
 JS_END;
-		wp_add_inline_script( 'jetpack-calendly-external-js', sprintf( $script, esc_url( $url ), esc_js( $block_id ) ) );
+		wp_add_inline_script(
+			'jetpack-calendly-external-js',
+			sprintf(
+				$script,
+				wp_json_encode(
+					esc_url_raw( $url ),
+					JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP
+				),
+				wp_json_encode( $block_id, JSON_HEX_TAG | JSON_HEX_AMP )
```

Comment:
May as well use the same flags here
```suggestion
				wp_json_encode( $block_id, JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP )
```

Reply:
Sure, done 👍